mutiny-core/src/lib.rs

```diff
@@ -404,22 +405,31 @@ impl<S: MutinyStorage> MutinyWallet<S> {
     /// Get contacts from the given npub and sync them to the wallet
     pub async fn sync_nostr_contacts(
         &self,
+        primal_url: Option<&str>,
         npub: XOnlyPublicKey,
-        timeout: Option<Duration>,
     ) -> Result<(), MutinyError> {
-        let keys = Keys::from_public_key(npub);
-        let options = Options::new().req_filters_chunk_size(30);
-        let client = Client::with_opts(&keys, options);
-
-        #[cfg(target_arch = "wasm32")]
-        client.add_relay("wss://relay.damus.io").await?;
-
-        #[cfg(not(target_arch = "wasm32"))]
-        client.add_relay("wss://relay.damus.io", None).await?;
-
-        client.connect().await;
+        let body = json!(["contact_list", {"pubkey": npub, "extended_response": true }]);
+
+        let url = primal_url.unwrap_or("https://dev.primal.net/api");
```

Comment:
Needs to point to our own.

Reply:
Fixed, it is still syncing events so might be a bit